tests/andromede/test_quota_co2.py

```diff
@@ -0,0 +1,194 @@
+from andromede.expression import literal, param, var
```

Comment:
Add the open source header to the file (you can paste it from any other Python file of the project)